File: src/com/codingbaby/ohmyidea/action/ui/ShowUIToggleActions.java
package com.codingbaby.ohmyidea.action.ui;

import com.codingbaby.ohmyidea.KeyHandler;
import com.intellij.openapi.actionSystem.AnAction;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.project.DumbAware;

/**
 * Created by baby on 16/11/6.
 */
public class ShowUIToggleActions extends AnAction implements DumbAware {


    @Override
    public void actionPerformed(AnActionEvent anActionEvent) {
        KeyHandler.executeAction("ViewToolBar",anActionEvent.getDataContext());
        KeyHandler.executeAction("ViewToolButtons",anActionEvent.getDataContext());
        KeyHandler.executeAction("ViewStatusBar",anActionEvent.getDataContext());
    }
}

File: src/main/java/com/codingbaby/ohmyidea/action/ui/ShowUIToggleActions.kt
package com.codingbaby.ohmyidea.action.ui

import com.codingbaby.ohmyidea.KeyHandler
import com.intellij.openapi.actionSystem.AnActionEvent
import com.intellij.openapi.project.DumbAwareAction

/**
 * Created by baby on 16/12/1.
 */
class ShowUIToggleActions : DumbAwareAction() {

    override fun actionPerformed(e: AnActionEvent) {
        KeyHandler.executeAction("ViewToolBar", e.getDataContext());
        KeyHandler.executeAction("ViewToolButtons", e.getDataContext());
        KeyHandler.executeAction("ViewStatusBar", e.getDataContext());
    }
}